SpeechResponder/SpeechResponder.cs

```diff
@@ -136,16 +136,12 @@ public void Handle(Event theEvent)
                 }
             }
 
-            IDictionary<string, object> data = Deserializtion.DeserializeData(theEvent.raw);
-            object val;
-
             if (theEvent is NavBeaconScanEvent)
             {
-                data.TryGetValue("NumBodies", out val);
-                beaconScanCount = (int)(long)val;
+                beaconScanCount = ((NavBeaconScanEvent)theEvent).numbodies;
                 Logging.Debug("beaconScanCount = " + beaconScanCount.ToString());
```

Comment:
ToString() is implict in a string concatentation, so you can omit it.
Better yet is string interpolation: prefix the string with a `$` and interpolate expressions in `{}`
`$"beaconScanCount = {beaconScanCount}"`